daemon/jitter_buffer.c

```diff
@@ -0,0 +1,335 @@
+#include "jitter_buffer.h"
+#include "media_socket.h"
+#include "media_player.h"
+#include "call.h"
+
+#define CONT_SEQ_COUNT 0x64
+#define CONT_MISS_COUNT 0x0A
+#define INITIAL_PACKETS 0x1E
+#define CONT_INCORRECT_BUFFERING 0x14
+
+
+static struct jitter_buffer_config *jb_config; 
+
+struct jitter_buffer_config {
+	int    min_jb_len;
+	int    max_jb_len;
+	int    enable_jb;
+	int    src_clockrate;
```

Comment:
I'm a bit confused about this parameter as I don't really see it being used, other than as an enable switch for the clock drift calculation.